gtsofa/medium-api
# #166213696 Set up test functionality for the application

### What is this PR supposed to do?
- Add spec rails a testing framework
- Add a library for faking data
- add a database cleaner
- add should matches
- add configurations to rails helper
### How should this be manually tested?
- clone the repo and navigate to the project directory
- check out to  this `ch-set-up-app-test-166213696` branch
- run Rspec

### Relevant PT stories
- [166213696](https://www.pivotaltracker.com/n/projects/2350589/stories/166213696)
